samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java

```diff
@@ -491,14 +491,16 @@ public boolean checkPublishedSettings(PublishedAssessmentService assessmentServi
 				String  date_err=ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.AssessmentSettingsMessages","date_error");
 				context.addMessage(null,new FacesMessage(date_err));
 			}
-			else if(!assessmentSettings.getIsValidFeedbackDate()){
-				String feedbackDateErr = ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.GeneralMessages","invalid_feedback_date");
-				context.addMessage(null,new FacesMessage(feedbackDateErr));
-				error=true;
+			else {
+				if(assessmentSettings.getFeedbackDate() == null && StringUtils.isNotBlank(assessmentSettings.getFeedbackEndDateString()) && assessmentSettings.getFeedbackDate().after(assessmentSettings.getFeedbackEndDate())){
```

Comment:
If getFeedbackDate is null, then using it later in the same if statement will give an NPE.

Reply:
Same that before. If getFeedbackDate is null then enters on if Line 489 and skip this else.